config.rb.in

```diff
@@ -0,0 +1,63 @@
+module BuildingSync
+
+  #ENV['GEM_HOME'] = 'C:/bricr/gems'
+  #ENV['GEM_PATH'] = 'C:/bricr/gems'
+  
+  # location of openstudio CLI
+  if /mswin/.match(RUBY_PLATFORM) || /mingw/.match(RUBY_PLATFORM)
+    OPENSTUDIO_EXE = 'C:/openstudio-2.6.0/bin/openstudio.exe'
+  else
+    OPENSTUDIO_EXE = '/usr/local/bin/openstudio'
+  end
+
+  # one or more measure paths
+  OPENSTUDIO_MEASURES = [
+      File.realpath("./measures", File.dirname(__FILE__)),
+      File.realpath("./../openstudio-measures/nrel_published",
+      File.dirname(__FILE__)),
+      File.realpath("./../openstudio-measures/nrel_dev", File.dirname(__FILE__))
+  ]
+
+  # one or more file paths
+  OPENSTUDIO_FILES = [
+      File.realpath("./weather",
+      File.dirname(__FILE__))
+  ]
+
+  # max number of datapoints to run
+  MAX_DATAPOINTS = Float::INFINITY
```

Comment:
Some of these number of simulation variables are not yet covered by the extension gem but they should be.

Reply:
I created a new issue for this #13